Tools/clang-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Can you please add a comment here that this is specific for Raspberry Pi? Otherwise the file is a bit lost.